src/pages/User/components/PullRequests/UserInfo/UserImage.js

```diff
@@ -9,7 +9,7 @@ const UserImage = ({ userImage, username }) => (
   >
     <img
       id="userImage"
-      alt={`GitHub profile image for ${username}`}
+      alt={`GitHub profile img for ${username}`}
```

Comment:
let's not change this. `image` is nicer

Reply:
Due to the resolve the warning message I had changed this to img from image